desktop/linux/release-notes/index.md

```diff
@@ -10,6 +10,21 @@ This page contains information about the new features, improvements, known issue
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.
 
+## Docker Desktop 4.8.2
+2022-05-09
+
+> Download Docker Desktop
+>
+> [DEB](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.8.2-amd64.deb) |
+> [RPM](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.8.2-x86_64.rpm) |
+> [Arch package](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.8.2-x86_64.pkg.tar.zst)
+
+### Bug fixes and minor changes
+
+- Fixed an issue with manual proxy settings which caused problems when pulling images. Fixes [docker/for-win#12714](https://github.com/docker/for-win/issues/12714).
+- Fixed high CPU usage when extensions are disabled. Fixes [docker/for-mac#6310](https://github.com/docker/for-mac/issues/6310).
```

Comment:
Would remove as only applicable to Mac

Reply:
@gtardif was this Mac only? I’m pretty sure I have seen reports for Windows as well on for-win.

Reply:
I believe this issues are applicable to all platforms, but were only reported on one issue tracker.